app/renderer/components/common/contextMenu.js

```diff
@@ -233,11 +233,11 @@ class ContextMenuSingle extends ImmutableComponent {
     if (this.props.y) {
       styles.marginTop = this.props.y
     }
-    const visibleMenuItems = this.props.template.filter((element) => {
+    const visibleMenuItems = this.props.template ? this.props.template.filter((element) => {
       return element.has('visible')
         ? element.get('visible')
         : true
-    })
+    }) : new Immutable.List()
```

Comment:
before this fix, the browser would stop working after you opened a bookmark folder